docs/wiki/Whats-new.md

```diff
@@ -2,23 +2,91 @@
 
 - [In this Section](#in-this-section)
 - [Updates](#updates)
+  - [December 2022](#december-2022)
+    - [Policy](#policy)
   - [November 2022](#november-2022)
+    - [Docs](#docs)
+    - [Tooling](#tooling)
+  - [Policy](#policy-1)
+  - [Other](#other)
```

Comment:
@Springstone can you roll back the changes to the TOC in this page, apart from the new month of December being added?

This is normally a VSCode extension that does this like MarkdownLint etc. (disabling it normally fixes it)

Reply:
Done @jtracey93 - had some PEBKAC this morning... sorry for the commits.